R/step-call-pivot_wider.R

```diff
@@ -75,6 +75,7 @@ pivot_wider.dtplyr_step <- function(data,
   } else {
     new_vars <- unique(pull(data, !!sym(names_from)))
   }
+  new_vars <- as.character(new_vars)
```

Comment:
The new line can be moved inside the "else" condition - when `length(names_from) > 1` the `paste()` call will already have `new_vars` as character.